kratos/python_scripts/point_output_process.py

```diff
@@ -59,6 +60,13 @@ def __init__(self, model, params):
         self.historical_value = self.params["historical_value"].GetBool()
         self.search_tolerance = self.params["search_tolerance"].GetDouble()
 
+        if self.params["search_configuration"].GetString() == "Initial":
+            self.search_configuration = KratosMultiphysics.Configuration.Initial
+        elif self.params["search_configuration"].GetString() == "Current":
```

Comment:
```suggestion
        if self.params["search_configuration"].GetString() == "initial":
            self.search_configuration = KratosMultiphysics.Configuration.Initial
        elif self.params["search_configuration"].GetString() == "current":
```

we use lower_case in json (I think is also written in the style guide)